src/main/java/seedu/address/model/person/Person.java

```diff
@@ -6,31 +6,45 @@
 import java.util.Objects;
 import java.util.Set;
 
+import javafx.beans.property.ObjectProperty;
+import javafx.beans.property.SimpleObjectProperty;
 import seedu.address.model.tag.Tag;
 import seedu.address.model.tag.UniqueTagList;
 
 /**
  * Represents a Person in the address book.
  * Guarantees: details are present and not null, field values are validated, immutable.
+ * Two persons can have the same profilePicture
  */
-public class Person {
+public class Person implements ReadOnlyPerson {
```

Comment:
There's a reason why we remove the `ReadOnlyPerson` interface. How come you bring it back again?